src/jinja2/ext.py

```diff
@@ -200,23 +227,40 @@ def _install(self, translations, newstyle=None):
         ngettext = getattr(translations, "ungettext", None)
         if ngettext is None:
             ngettext = translations.ngettext
-        self._install_callables(gettext, ngettext, newstyle)
+
+        pgettext = translations.pgettext
+        npgettext = translations.npgettext
```

Comment:
I think we need to use `getattr` to fall back to `None` here for full backwards compatibility

Reply:
Add